test-data/unit/check-commented-type-annotations.test

```diff
@@ -0,0 +1,40 @@
+[case testCommentedTypeAnnotations]
+# flags: --allow-commented-type-annotations
+
+class A():
+    b = ""
+# type: str
+
+
+[case testIncorrectTypeAnnotations]
+# flags: --allow-commented-type-annotations
+
+class A():
+    b: int = ""
+# type: str
+[out]
+main:4: error: Incompatible types in assignment (expression has type "str", variable has type "int")
+
+[case testCommentedTypeAnnotationsWithVal]
+# flags: --allow-commented-type-annotations
+
+# type: str = "hello"
```

Comment:
I'm not sure what this is supposed to test. `# type: str = "hello"` is not a valid type annotation comment.